test/tests/mail.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const Mail = require('../../lib/mail');
+
+describe('Mail', ()=> {
+  describe('Mail properties', ()=> {
+    let mail;
+
+    beforeEach((done) => {
+      mail = new Mail('ssing128@gmail.com', 'ssing128@gmail.com', 'i love you');
+      mail.send(()=> {
+        console.log('__calling done')
+        done();
+      })
+    })
+
+    it('should set fromEmail', ()=> {
```

Comment:
These tests aren't really testing anything. You're testing mongoose code, thus, they should not be here at all.

Reply:
I have removed this file from test because i will rewrite the tests.